### ISSUE TITLE
Tile Select Group and Tile Select - Add `aria-hidden` attr to checkbox SVG - (2036570357)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5609]
### Module: 
 34 Tile Select Group and Tile Select 

### Violation: 
 Provide alternative text for images 

![image](https://user-images.githubusercontent.com/5023024/202728840-d47b19a8-4bd4-40a4-9f1e-59feba6bdf3f.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036570357 

### Severity: 
 10 

### Description: 
 [Issue]
There are checkbox images that serve a decorative purpose but are exposed to screen readers.

[User Impact]
Screen readers may announce SVGs with no accessible name as "group." This gives screen reader users irrelevant and confusing information.

[Code Reference] 
 ```
<svg class="check-svg" viewbox="0 0 16 16">
  <path d=""></path>
</svg>
 ``` 

### Note: 
 [Suggestion]
Provide alternative text for images. Non-meaningful images must be marked as decorative, which causes screen readers to ignore them. To mark an ``` <svg> ``` element as decorative, add aria-hidden="true" to the element.

[Compliant Code Example] 
 ```
<svg aria-hidden="true" class="check-svg" viewbox="0 0 16 16">
  <path d=""></path>
</svg>
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Will be a fix similar to https://github.com/Esri/calcite-components/pull/5211

**Comment by geospatialem** [comment]
Verified in `beta.99`. Fix from https://github.com/Esri/calcite-components/pull/6069.
